.mocharc.json

```
@@ -0,0 +1,5 @@
+{
+  "require": ["dotenv/config", "ts-node/register", "source-map-support/register"],
```

```suggestion
  "require": "ts-node/register/files",
```

And also delete the `.env` file and add it to the `.gitignore`?